Makefile

```diff
@@ -33,4 +33,7 @@ test-cleanup:
 	@cd tests/e2e; rm -rf local-plugins .terraform .terraform.lock.hcl terraform.tfstate terraform.tfstate.backup
 
 acceptance:
-	@TF_ACC=1 go test -v ./tests/resources
\ No newline at end of file
+	@TF_ACC=1 go test -v ./tests/resources
+	
+test-unit:
```

Comment:
I'm using build tags to run unit tests separately - so that any code that is not generated can have unit tests in future.
Callout - I've missed adding this step in the github workflow, will do.

Reply:
Unit tests are now part of workflow [Terraform Provider Tests / Unit Tests (Terraform 1.11.*) (pull_request)](https://github.com/Kong/terraform-provider-konnect/actions/runs/14753292976/job/41415508167?pr=173)